tests/cli.rs

```diff
@@ -1,191 +1,197 @@
-
-    assert_cli::Assert::command(&[&BIN, "build", "-L", "debug", "-d", &dest_param])
-        .current_dir(CWD.join("tests/fixtures/example"))
-        .stderr()
-        .doesnt_contain("[trace]")
-        .stderr()
-        .contains("[debug]")
-        .stderr()
-        .contains("[info]")
+    let project_root = assert_fs::TempDir::new().unwrap();
+    project_root
+        .copy_from("tests/fixtures/example", &["*"])
         .unwrap();
 
-    destdir.close().unwrap();
+    process::Command::cargo_bin("cobalt")
+        .unwrap()
+        .args(&["build", "-L", "debug"])
+        .current_dir(project_root.path())
+        .assert()
+        .success()
+        .stderr(&predicate::str::contains("[trace]").not().from_utf8())
+        .stderr(&predicate::str::contains("[debug]").from_utf8())
+        .stderr(&predicate::str::contains("[info]").from_utf8());
+
+    project_root.close().unwrap();
 }
 
 #[test]
 pub fn log_levels_info() {
-    let destdir = TempDir::new("info").expect("Tempdir not created");
-    let dest_param = destdir
-        .path()
-        .to_str()
-        .expect("Can't convert destdir to str")
-        .to_owned();
-
-    assert_cli::Assert::command(&[&BIN, "build", "-L", "info", "-d", &dest_param])
-        .current_dir(CWD.join("tests/fixtures/example"))
-        .stderr()
-        .doesnt_contain("[trace]")
-        .stderr()
-        .doesnt_contain("[debug]")
-        .stderr()
-        .contains("[info]")
+    let project_root = assert_fs::TempDir::new().unwrap();
+    project_root
+        .copy_from("tests/fixtures/example", &["*"])
         .unwrap();
 
-    destdir.close().unwrap();
+    process::Command::cargo_bin("cobalt")
+        .unwrap()
+        .args(&["build", "-L", "info"])
+        .current_dir(project_root.path())
+        .assert()
+        .success()
+        .stderr(&predicate::str::contains("[trace]").not().from_utf8())
+        .stderr(&predicate::str::contains("[debug]").not().from_utf8())
+        .stderr(&predicate::str::contains("[info]").from_utf8());
+
+    project_root.close().unwrap();
 }
 
 #[test]
 pub fn log_levels_silent() {
-    let destdir = TempDir::new("silent").expect("Tempdir not created");
-    let dest_param = destdir
-        .path()
-        .to_str()
-        .expect("Can't convert destdir to str")
-        .to_owned();
-
-    assert_cli::Assert::command(&[&BIN, "build", "--silent", "-d", &dest_param])
-        .current_dir(CWD.join("tests/fixtures/example"))
-        .stderr()
-        .is("")
-        .stdout()
-        .is("")
+    let project_root = assert_fs::TempDir::new().unwrap();
+    project_root
+        .copy_from("tests/fixtures/example", &["*"])
         .unwrap();
 
-    destdir.close().unwrap();
+    process::Command::cargo_bin("cobalt")
+        .unwrap()
+        .args(&["build", "--silent"])
+        .current_dir(project_root.path())
+        .assert()
+        .success()
+        .stdout(&predicate::str::is_empty().from_utf8())
```

Comment:
Can't move forward with this until https://github.com/assert-rs/predicates-rs/issues/7